frontend/app/professor/simulation-builder/page.tsx

```
@@ -695,7 +792,327 @@ export default function ScenarioBuilder() {
+    case 'title':
+      setName(fieldValue);
+      // Update database completion field
+      setDbCompletionFields(prev => ({
+        ...prev,
+        nameCompleted: !!fieldValue?.trim()
+      }));
+      markAsUnsaved();
+      break;
+    case 'description':
+      const formattedDescription = formatDescription(fieldValue);
+      setDescription(formattedDescription);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)